### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 11

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#276] [Esercizi - Viaggi/Piazza]
**Esercizio 5.7 del libro di testo (Geometria Analitica: Marco Abate & Chiara De Fabritiis)**

**Comment by daromacarlo** [comment]
![Immagine 2023-12-04 175302](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/001a853a-36f5-491d-a51e-eb24f53cdcf7)
